lib/fsAsync.js

```diff
@@ -85,6 +82,22 @@ function readdirRecursive(options) {
   });
 }
 
+// Options contains a path and optionally files to ignore.
+// @param options.path the directory to recurse
+// @param options.ignore files to ignore
+// @returns array of {name, mode} for files that match
+function readdirRecursive(options) {
+  var mmopts = { matchBase: true, dot: true };
```

Comment:
Is there a reason why you chose these options?

Reply:
They were the options that fstreamIgnore hard coded.